### ISSUE TITLE
kuka_plan_runner warning every time I run bazel

### ISSUE THREAD

**Issue by RussTedrake** [RobotLocomotion/drake#7144]
I believe this is related to the recent goal of exposing kuka_plan_runner to spartan, but now I see this warning every time I run bazel:
```
WARNING: /home/russt/.cache/bazel/_bazel_russt/fee0ea1b9d6c599156f0d1f0dd11b82e/external/drake/tools/install.bzl:62:9: //drake/examples/kuka_iiwa_arm:install installing file tools/install/kuka_plan_runner/kuka_plan_runner.sh which is not in current package.
```

**Comment by fbudin69500** [comment]
Thanks for the report. This problem will be addressed in https://reviewable.io/reviews/robotlocomotion/drake/6772 .

**Comment by fbudin69500** [comment]
Fixed in #6772
